src/connectors/copyparty.ts

```diff
@@ -0,0 +1,17 @@
+export {};
+
+Connector.playerSelector = '#np_inf';
+Connector.pauseButtonSelector = '#np_inf.playing';
+
+Connector.trackArtSelector = '#np_img';
+Connector.albumArtistSelector = '#np_circle';
+Connector.albumSelector = '#np_album';
+Connector.artistSelector = '#np_artist';
+Connector.trackSelector = '#np_title';
+
+Connector.currentTimeSelector = '#np_pos';
+Connector.durationSelector = '#np_dur';
+
+this.getUniqueID = () => {
+	return document.getElementById('np_url').textContent;
```

Comment:
Could this just use `Util.getTextFromSelectors('#np_url')` instead?

Reply:
Absolutely, wasn't aware of that helper. Will fix that 👍